make/Main.gmk

```diff
@@ -508,13 +508,13 @@ $(eval $(call SetupTarget, docs-jdk-index, \
 $(eval $(call SetupTarget, docs-zip, \
     MAKEFILE := Docs, \
     TARGET := docs-zip, \
-    DEPS :=  docs-jdk, \
+    DEPS :=  docs-jdk buildtools-jdk, \
```

Comment:
Is this really needed? I did not think jrtfs-jar used zip?

Reply:
Sorry, I'm mis-reading the lines here. It's of course for docs-zip. Then it's perfectly in order! :-)

Reply:
It's using JarArchive.gmk, which is a similar, but different and probably also needs the same treatment. We should probably share more code between them.

Reply:
I believe so, it's target docs-zip not jrtfs : https://github.com/openjdk/jdk/blob/51a5731d6dc4b6f6feac920a4b8b49c15fd6b34f/make/Docs.gmk#L712